File: Quid_pro_Quo/WebApiModels/ExchangeOfServicesApiModel.cs
using System;

namespace Quid_pro_Quo.WebApiModels
{
    public class ExchangeOfServicesApiModel
    {
        public int RequestingPostId { get; set; }
        public int RequestedPostId { get; set; }
        public int DateNumberOfRequestedPost { get; set; }
        public int DateNumberOfRequestingPost { get; set; }
        public string Text { get; set; }
        public DateTime Time { get; set; }
    }
}

File: Web/Quid_pro_Quo/ClientApp/src/app/models/exchange-of-services-api.model.ts
export interface ExchangeOfServicesApiModel {
  id: number;
  requestingPostId: number;
  requestedPostId: number;
  dateNumberOfRequestingPost: number;
  dateNumberOfRequestedPost: number;

  text: string;
  time: Date;

  proposalStatus: StatusEnum;
  doneStatus1: StatusEnum;
  doneStatus2: StatusEnum;
  notViewed: boolean;
}

export enum StatusEnum {
  noInfo = 0,
  yes = 1,
  no = 2,
}
